doc/nvim-tree-lua.txt

```diff
@@ -1545,6 +1545,15 @@ node.open.vertical()                      *nvim-tree-api.node.open.vertical()*
 node.open.horizontal()                  *nvim-tree-api.node.open.horizontal()*
     |nvim-tree-api.node.edit()|, file will be opened in a new horizontal split.
 
+node.open.drop()                               *nvim-tree-api.node.open.tab()*
+    Switch to window with selected file if it exists.
+    Open file in current window otherwise.
```

Comment:
This is strictly correct, however `view._prevent_buffer_override()` will always fire to eject the newly opened file.

How about "Open file otherwise"?

Reply:
Updated, good suggestion, thanks.